ingestors/support/convert.py

```diff
@@ -15,6 +15,7 @@
 CONVERT_DIR = os.path.join(gettempdir(), "convert")
 OUT_DIR = os.path.join(CONVERT_DIR, "out")
 TIMEOUT = 3600
+CONVERT_RETRIES = 10
```

Comment:
I'm curious if the results will ever really change after 3 or 4 tries

Reply:
They tend not to, indeed. We're coming down from 256 retries. Might settle on 5.

Reply:
I have looked and 5 or 6 seems like a nice number.